delivery_location/tests/test_deliverylocation.py

```diff
@@ -0,0 +1,112 @@
+import pytest
+from delivery_location.models import Supplier, DeliveryLocation
+from django.core.exceptions import ValidationError
+import datetime
+
+
+@pytest.fixture
+def TEST_DATA():
```

Comment:
now that it's a fixture, it needs to be in lowercase. Sorry if I misled you.

Reply:
Its okay,its a new thing to learn
Fixed!